hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -431,6 +431,15 @@
                rel_path: "hw/top_earlgrey/ip_autogen/alert_handler/{sub_flow}/{tool}"
                cov: false
              } */
+             {
+               name: clkmgr_sec_cm
+               dut: clkmgr
+               fusesoc_core: lowrisc:dv:clkmgr_sva
+               import_cfgs: ["{proj_root}/hw/formal/tools/dvsim/common_fpv_cfg.hjson"]
+               rel_path: "hw/ip/clkmgr/{sub_flow}/{tool}"
+               cov: false
```

Comment:
Just out of curiosity, what cov:false means here?

Reply:
hi Jaedon, 
we do not collect any code coverage here because this test is not intended to cover all functions for clkmgr, but just make sure all sec_cm related assertions pass unconditionally in formal env.

Reply:
> 

gotcha. thanks